### PR TITLE
[GSOC] 1st objective STARDIS Parallelization and GPU

### PR DESCRIPTION
**Pull request by zafar-hussain**
### :pencil: Description

**Type:** : `documentation` 

A Numba compiled version of planck_function() was created, and a comparison of how long it takes to calculate planck_function(nus_planck) for the original function versus the numba jitted function for 100, 200, and 500 frequency bins was done.

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [x] Other method (new function is tested with the old function on the code)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

### CODE REVIEW

**Comment by AyushiDaksh** [comment]
hi @zafar-hussain , Can you please reformat your title to have "[GSoC]" (like other GSoC requests) at the beginning for ease of search?

File: docs/physics/montecarlo/initialization.ipynb [hunk 1]
```diff
@@ -71,10 +71,39 @@
   },
```
Comment (smithis7) [inline docs/physics/montecarlo/initialization.ipynb:71]: <p>This looks really good. One thing that would be great to show in this is perhaps the compilation time penalty for the first time you run this. Of course, you can only show this once in the notebook, but even comparing the first and second time you run the timing would show it (you may need to lower the number runs/loops)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (smithis7) [inline docs/physics/montecarlo/initialization.ipynb:71]: <p>It all looks really good -- I would suggest trying this without a <code>lambda</code> function, named functions should use <code>def</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (zafar-hussain) [reply docs/physics/montecarlo/initialization.ipynb:71]: Hi @smithis7 ,

due to my mistake, this PR was closed, 

please have a look at PR #2254, I have updated the Initialization notebook, in accordance to your suggestions

cheers

**Pull request closed without merging**